### PR TITLE
Kalyan fix device port

### PR DESCRIPTION
**Pull request by kalyan-nidumolu**
Obsolete redundant fields in DevicePort

### CODE REVIEW

File: api/proto/info/info.proto [hunk 1]
```diff
@@ -298,7 +301,6 @@ message DevicePort {
   ZInfoDNS dns = 25; // If DHCP assigned
   bool up = 26;      // operational up/down status.
   GeoLoc location = 27;
-  bool uplink = 28; // Uplink interface  // XXX rename to isMgmt
```
Comment (eriknordmark) [inline api/proto/info/info.proto]: Have you checked whether zedcloud references this field?

File: api/proto/info/info.proto [hunk 2]
```diff
@@ -284,9 +284,12 @@ message DevicePort {
   // DhcpConfig
   uint32 dhcpType = 11;
   string subnet = 12;
+  // gateway - OBSOLETE - obsoleted by defaultRouters
   string gateway = 13;
+  // domainname - OBSOLETE - obsoleted by dns
```
Comment (eriknordmark) [inline api/proto/info/info.proto]: Isn't dns about about DHCP derived status, and this domainname and dnsServers is about static config? I honestly don't know the answer.

File: api/proto/info/info.proto (hunk 1 above)
Comment (kalyan-nidumolu) [reply api/proto/info/info.proto]: This is new. Zedcloud doesn't use it

File: api/proto/info/info.proto (hunk 2 above)
Comment (kalyan-nidumolu) [reply api/proto/info/info.proto]: I see this information is being filled from NetworkPortConfig.. Not from Status..

**Pull request merged**